models/general/entertainment/en/cooking_guide.txt

```diff
@@ -1,13 +1,15 @@
 #Gives recipies and links to cook a dish
 * cook *
 !console:To cook  $title$ , make sure you have $ingredients$ and check out $href$
+!example:how cook biryani
```

Comment:
Writing the !example after !console doesn't let the skill work. You can test it using [this](http://dream.susi.ai/p/coo) example.